### PR TITLE
Fix/gestures no tap coordinate

### PR DESCRIPTION
**Pull request by TobiasRoikjer**

### CODE REVIEW

File: lib/calabash/device.rb [hunk 1]
```diff
@@ -174,9 +170,7 @@ def long_press(query, options={})
       gesture_options[:at] ||= {}
       gesture_options[:at][:x] ||= 50
       gesture_options[:at][:y] ||= 50
-      gesture_options[:offset] ||= {}
-      gesture_options[:offset][:x] ||= 0
-      gesture_options[:offset][:y] ||= 0
+      gesture_options[:offset] = {x: 0, y: 0}
```
Comment (jmoody) [inline lib/calabash/device.rb]: @TobiasRoikjer Did you leave this in for iOS?

Comment (TobiasRoikjer) [inline lib/calabash/device.rb]: I thought we might reuse it some day

**Comment by jmoody** [comment]
@TobiasRoikjer I pulled this.  I think I need to rebase and force push.

I will be take over from here.

**Comment by jmoody** [comment]
Actually, let's just merge this and I will make a new PR.  Thanks.

**Pull request merged**